src/components/__tests__/source-info.test.jsx

```diff
@@ -0,0 +1,58 @@
+/* eslint-env jest */
+
+import React from 'react';
+import {render} from 'enzyme';
```

Comment:
Is there a reason you’re using the static renderer instead of the `shallow` or `mount`? I don’t think we use it elsewhere and the other two have a much more full API, letting you write nicer selectors and assertions, like:

```js
const link = sInfo.find('a').at(0);
expect(link.prop('href')).toBe('https://web.archive.org/web/*/https://19january2017snapshot.epa.gov/test-url');
```

Reply:
Err...I think the reason is because I'm an enzyme newb 😅I was actually wondering why `.at` wasn't working...heh. Seems like `shallow` should be sufficient for this test case.

Reply:
No worries, I find Enzyme to be pretty confusing a lot of the time, too. Too many ways to render something and too easy to confuse what’s happening.